apps/storefront/app/routes/playground.tsx

```diff
@@ -1,6 +1,18 @@
+import type { MetaFunction } from '@remix-run/node';
+
 import { Heading } from '@digdir/designsystemet-react';
 import PlaygroundGUI from '../components/PlaygroundGUI/PlaygroundGUI.client';
 
+export const meta: MetaFunction = () => {
+  return [
+    { title: 'Organisation Chart' },
```

Comment:
```suggestion
    { title: 'Lekeplass - Organisation Chart' },
```
Suggest adding the title for the page